tests/test_utf8_chunk.py

```diff
@@ -8,27 +8,27 @@ class TestUtf8Chunk(unittest.TestCase):
     def test_noop_ascii(self):
         """utf8_chunk returns input bytes if byte sequence is entirely ASCII"""
         bytestr = b'hello world'
-        self.assertIs(utf8_chunk(bytestr), bytestr)
+        self.assertTrue(utf8_chunk(bytestr) is bytestr)
 
     def test_noop_utf8_end(self):
         """utf8_chunk returns input bytes if byte sequence uses non-ASCII
         UTF-8 at the end of the string and is well-formed"""
         unistr = u'hello 世界'
         bytestr = unistr.encode('utf-8')
-        self.assertIs(utf8_chunk(bytestr), bytestr)
+        self.assertTrue(utf8_chunk(bytestr) is bytestr)
 
     def test_noop_utf8_mid(self):
         """utf8_chunk returns input bytes if byte sequence uses non-ASCII
         UTF-8 in the middle of the string and is well-formed"""
         unistr = u'hello 世界!'
         bytestr = unistr.encode('utf-8')
-        self.assertIs(utf8_chunk(bytestr), bytestr)
```

Comment:
Yes, exactly.  That's what the method is documented as doing.  (The chunks could be several MB in size and the purpose of the patch series is to reduce memory usage so it seems prudent to avoid unnecessarily copies.)